jdaviz/configs/default/plugins/viewers.py

```diff
@@ -174,10 +174,15 @@ def _on_subset_create(self, msg):
         if msg.subset.label not in self._expected_subset_layers and msg.subset.label:
             self._expected_subset_layers.append(msg.subset.label)
 
-    def _initialize_toolbar_nested(self, default_tool_priority=[]):
-        # would be nice to call this from __init__,
-        # but because of inheritance order that isn't simple
-        self.toolbar_nested = NestedJupyterToolbar(self, self.tools_nested, default_tool_priority)
+    def initialize_toolbar(self, default_tool_priority=[]):
+        # NOTE: this overrides glue_jupyter.IPyWidgetView
+        self.toolbar = NestedJupyterToolbar(self, self.tools_nested, default_tool_priority)
+
+    @property
+    def tools(self):
```

Comment:
Coverage says this is not tested. Is this really used?

Reply:
we don't use it anywhere, but glue-jupyter defaults `viewer.tools` to a list of strings for what `BasicJupyterToolbar` would use.  Since this PR now prevents `BasicJupyterToolbar` from being created and replaces it with `NestedJupyterToolbar`, I figured we shouldn't carry around the irrelevant list and so just replace it with the equivalent flat list of tool names in our toolbar.

Reply:
With the upstream fix, do we still need this?

Reply:
Yes, we still have our own toolbar implementation.